package.json

```diff
@@ -1,8 +1,27 @@
 {
-  "private": true,
+  "name": "etcd-website",
+  "version": "0.0.1",
+  "description": "etcd website.",
+  "main": "none.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/etcd-io/website.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/etcd-io/website/issues"
+  },
+  "homepage": "https://etcd.io",
```

Comment:
AFAIK, we don't need these entries since the website isn't going to be published as a package.